src/main/java/net/javadiscord/javabot/listener/HugListener.java

```diff
@@ -62,54 +94,9 @@ public void onMessageReceived(@Nonnull MessageReceivedEvent event) {
 		String content = event.getMessage().getContentRaw();
 		if (FUCKER.matcher(content).find()) {
 			long threadId = event.isFromThread() ? event.getChannel().getIdLong() : 0;
-			WebhookUtil.ensureWebhookExists(tc,
-					wh -> sendWebhookMessage(wh, event.getMessage(), replaceFucks(content), threadId),
-					e -> ExceptionLogger.capture(e, getClass().getSimpleName()));
+			WebhookUtil.ensureWebhookExists(tc, wh -> WebhookUtil.replaceMemberMessage(wh, event.getMessage(), replaceFucks(content), threadId), e -> ExceptionLogger.capture(e, getClass().getSimpleName()));
```

Comment:
Put the arguments on different lines because that allows to immediately differentiate between the lambdas.